package.json

```diff
@@ -18,45 +18,36 @@
   "author": "Steve Heffernan",
   "scripts": {
     "sandbox": "node build/sandbox.js",
-    "prestart": "npm-run-all sandbox",
-    "start": "npm-run-all -p watch karma-server",
+    "assets": "node build/assets.js",
+    "start": "npm-run-all -p sandbox watch karma-server",
     "clean": "shx rm -rf ./dist ./test/dist ./docs/api",
     "postclean": "shx mkdir -p ./dist/lang ./test/dist",
-    "changelog": "conventional-changelog -p videojs -i CHANGELOG.md -s",
+    "lint": "vjsstandard",
+    "lint-errors": "vjsstandard --errors",
+    "karma-server": "karma start test/karma.conf.js --singleRun=false --auto-watch",
     "build": "npm-run-all build-dev minify copy zip",
+    "copy": "npm-run-all copy:*",
+    "copy:fonts": "shx cp -R node_modules/videojs-font/fonts dist/font",
+    "copy:examples": "shx cp -R docs/examples dist/",
+    "zip": "cd dist && cross-env bestzip \"./video-js-${npm_package_version}.zip\" * && cd ..",
+
     "build-dev": "npm-run-all clean lint-errors build:js build:css build:lang build:test",
     "build:test": "npm-run-all build:test:*",
     "build:test:a11y": "shx cp sandbox/descriptions.html.example sandbox/descriptions.test-a11y.html",
     "build:test:browserify": "browserify test/require/browserify.js -o test/dist/browserify.js",
     "build:test:webpack": "webpack --hide-modules test/require/webpack.js test/dist/webpack.js",
-    "copy": "npm-run-all copy:*",
-    "copy:fonts": "shx cp -R node_modules/videojs-font/fonts dist/font",
-    "copy:examples": "shx cp -R docs/examples dist/",
     "build:js": "rollup -c",
-    "build:css": "npm-run-all build:css:*",
-    "build:css:cdn": "node-sass src/css/vjs-cdn.scss dist/alt/video-js-cdn.css",
-    "postbuild:css:cdn": "postcss --verbose --config postcss.config.js -d dist/alt dist/alt/video-js-cdn.css",
-    "build:css:default": "node-sass src/css/vjs.scss dist/video-js.css",
-    "postbuild:css:default": "postcss --verbose --config postcss.config.js -d dist/ dist/video-js.css",
+    "build:css": "node build/build-css.js",
     "build:lang": "npm-run-all build:lang:*",
     "build:lang:js": "vjslang --dir dist/lang",
     "build:lang:copy": "shx cp -R lang/* dist/lang/",
     "minify": "npm-run-all minify:*",
-    "minify:js": "babel-node build/minify.js",
```

Comment:
After the rollup tests pr is merged, I will work on a pr to use the shared rollup config, then the `minify` script will go away entirely.

Reply:
Having rollup minify significantly increased build times because it had to be a separate rollup build.

Reply:
now that terser supports `include/exclude` this won't be an issue.